ktor-features/ktor-i18n/jvm/src/io/ktor/i18n/I18n.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.i18n
+
+import io.ktor.application.*
+import io.ktor.request.*
+import io.ktor.util.*
+import io.ktor.util.pipeline.*
+import java.nio.charset.*
+
+/**
+ * Represents I18n feature and its configuration.
+ *
+ * install(I18n) {
+ *    defaultLanguage = "pt-BR"
+ *    encoding = StandardCharsets.UTF_8
+ * }
+ */
+class I18n(configuration: Configuration) {
+
+    private val defaultLanguage = configuration.defaultLanguage
+
+    private val encoding = configuration.encoding
+
+    /**
+     * I18n configuration. Currently supports [encoding] and [defaultLanguage]
+     *
+     * [defaultLanguage] must follow IETF BCP 47 language tag string specification
+     */
+    class Configuration {
+        var defaultLanguage: String = ""
+        var encoding: Charset = StandardCharsets.UTF_8
+    }
+
+    private fun intercept(context: PipelineContext<Unit, ApplicationCall>) {
+        val acceptedLanguage = context.call.request.acceptLanguage() ?: defaultLanguage
```

Comment:
Here we need to process a list of accepted languages. I suggest to use `acceptLanguageItems()` instead to get a sorted list of accepted languages provided by a client (sorted by quality)